pkg/lib/config/config.go

```diff
@@ -41,7 +41,8 @@ var _ = Schema.Add("AppConfig", `
 		"forgot_password": { "$ref": "#/$defs/ForgotPasswordConfig" },
 		"welcome_message": { "$ref": "#/$defs/WelcomeMessageConfig" },
 		"verification": { "$ref": "#/$defs/VerificationConfig" },
-		"otp": { "$ref": "#/$defs/OTPConfig" },
+		"sms": { "$ref": "#/$defs/SMSConfig" },
+		"email": { "$ref": "#/$defs/EmailConfig" },
```

Comment:
I just realized we have MessagingConfig. Please move SMSConfig and EmailConfig inside MessageConfig.